virtual_rainforest/models/soil/carbon.py

```diff
@@ -427,7 +528,10 @@ def calculate_direct_litter_input_to_lmwc(
             pools [kg C m^-2 day^-1].
 
     Returns:
-        Amount of carbon directly added to LMWC pool from litter.
+        Amount of carbon directly added to LMWC and POM pools from litter.
     """
```

Comment:
Will this differ between grid cells at some point?

Reply:
Yeah this is something that's a place holder until the `litter` module/model is added. Felt easier to keep this as a constant across all cells until a link with the litter module is made